src/renderer/styles/statetable.less

```diff
@@ -55,3 +55,49 @@
 .StateTable-Warning-Info {
   padding-right: 5rem;
 }
+
```

Comment:
question: is there a reason we have a mix of inline styles and added LESS styles in this PR?

Reply:
I think it'd be good to put all the styles into the stylesheet 👍